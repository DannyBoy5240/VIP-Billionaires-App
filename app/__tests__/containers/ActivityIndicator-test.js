import React from 'react'
import renderer from 'react-test-renderer'
import RCActivityIndicator from '../../containers/ActivityIndicator'

test('renders correctly', () => {
  const tree = renderer.create(<RCActivityIndicator styles={{}} theme={'dark'} absolute={true} />).toJSON()
  expect(tree).toMatchSnapshot()
})
