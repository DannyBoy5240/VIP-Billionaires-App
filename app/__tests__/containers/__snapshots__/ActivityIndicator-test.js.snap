// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ActivityIndicator
  color="#FFFFFF"
  style={
    Array [
      Object {
        "flex": 1,
        "padding": 16,
      },
      Object {
        "alignItems": "center",
        "bottom": 0,
        "justifyContent": "center",
        "left": 0,
        "position": "absolute",
        "right": 0,
        "top": 0,
        "zIndex": 100000,
      },
    ]
  }
  styles={Object {}}
/>
`;
